modules/appnexusBidAdapter.js

```diff
@@ -304,6 +304,13 @@ export const spec = {
       payload.us_privacy = bidderRequest.uspConsent;
     }
 
+    if (bidderRequest?.gppConsent) {
```

Comment:
If another adapter is implementing and only is interested in regs.gpp and regs.gpp_sid, they can also look at 
 bidderRequest?.ortb2?.regs?.gpp